mono_repo/lib/src/mono_config.dart

```diff
@@ -4,6 +4,8 @@
 
 import 'package:json_annotation/json_annotation.dart';
 import 'package:path/path.dart' as p;
+// ignore: implementation_imports
+import 'package:pubspec_parse/src/dependency.dart';
```

Comment:
Do we need to expose this? We could also do something funky where we create a fake pubspec and parse that to avoid the impl import?